amundsen_application/api/metadata/v0.py

```diff
@@ -553,3 +553,34 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+# TODO: Implement real support
+@metadata_blueprint.route('/user', methods=['GET'])
+def get_user() -> Response:
+    try:
+        user_id = get_query_param(request.args, 'user_id')
+        user_info = {
+            'first_name': 'Firstname',
+            'last_name': 'Lastname',
+            'email': 'test@test.com',
+            'display_name': 'Firstname Lastname',
+            'profile_url': 'https://github.com/lyft/amundsenfrontendlibrary',
+            'user_id': user_id,
+            'github_name': 'lyft',
+            'is_active': True,
+            'manager_name': 'Roald Amundsen',
+            'role_name': 'Software Engineer',
+            'slack_url': 'https://slack.com',
+            'team_name': 'Amundsen Team',
+        }
+        if user_id == 'alumni':
```

Comment:
why user_id will be alumni?

Reply:
This was just for testing so that I could type in `amundsen-url/user/alumni` in my dev environment to check the alumni version of the page.